sql/core/src/test/scala/org/apache/spark/sql/connector/DataSourceV2SQLSuite.scala

```diff
@@ -1799,6 +1799,18 @@ class DataSourceV2SQLSuite
     }
   }
 
+  test("DESCRIBE FUNCTION not valid v1 namespace") {
```

Comment:
the test name is a little confusing. How about `DESCRIBE FUNCTION: only support session catalog`